Podfile

```diff
@@ -44,6 +45,8 @@ end
 target :WordPressTest, :exclusive => true do
   pod 'OHHTTPStubs', '1.1.1'
   pod 'OCMock'
+  pod 'Specta'
```

Comment:
Done in https://github.com/wordpress-mobile/WordPress-iOS/commit/f626aa3dd02e05f21e61baa22a329868e448a4a9